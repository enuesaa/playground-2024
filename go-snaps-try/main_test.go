package main

import (
	"testing"

	"github.com/gkampitakis/go-snaps/snaps"
)
  
func TestHello(t *testing.T) {
	snaps.MatchSnapshot(t, hello())
}

func TestOther(t *testing.T) {
	snaps.MatchSnapshot(t, 5, 10, 20, 25)
}
