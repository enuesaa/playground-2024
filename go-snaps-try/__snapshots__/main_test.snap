
[TestHello - 1]
world
---

[TestOther - 1]
int(5)
int(10)
int(20)
int(25)
---
